backend/support/export_audit_streams.py

```diff
@@ -0,0 +1,169 @@
+import support.export_audit_sql as export_audit_sql
+from sling import ReplicationStream
+
+map_table_to_file = {
+    "additional_eins": "AdditionalEIN",
```

Comment:
actually, I think we want to map `additional_eins` to... `additional_eins`...

That is, for consistency with the API, lets not create new table names. This way, we can refer to the API data dictionary and say "the additional_eins files have the same dictionary as the additiona_eins API."

https://github.com/GSA-TTS/FAC/issues/4827

I added that *after* this work started, so it's totally me creating a spec after you did the work. :P Apologies.